.github/workflows/enforceTestCoverage.yml

```diff
@@ -19,11 +19,13 @@ jobs:
       contents: read
     env:
       ETH_NODE_URI_MAINNET: ${{ secrets.ETH_NODE_URI_MAINNET }}
-      ETH_NODE_URI_POLYGON: ${{ secrets.ETH_NODE_URI_POLYGON }}
-      ETH_NODE_URI_GOERLI: ${{ secrets.ETH_NODE_URI_GOERLI }}
+      # keep the rest in alphabetic order
       ETH_NODE_URI_ARBITRUM: ${{ secrets.ETH_NODE_URI_ARBITRUM }}
+      ETH_NODE_URI_BASE: ${{ secrets.ETH_NODE_URI_BASE }}
       ETH_NODE_URI_BSC: ${{ secrets.ETH_NODE_URI_BSC }}
       ETH_NODE_URI_GNOSIS: ${{ secrets.ETH_NODE_URI_GNOSIS }}
+      ETH_NODE_URI_GOERLI: ${{ secrets.ETH_NODE_URI_GOERLI }}
```

Comment:
Can you please validate if we are actually using `GOERLI` in our tests.

Reply:
We aren't. There are many related things to depricated `GOERLI` and` MUMBAI` networks. We decided to go with seperate ticket for that as it's bigger topic and it's not our prio at this moment.

 https://discord.com/channels/849912621360218112/1331512233183674480/1331617068583358586
 and created PR for it (currently in the backlog): https://lifi.atlassian.net/browse/LF-11911.
 
 In this PR I just reordered it to keep it clean.